service/controller/clusterapi/v27/key/key.go

```diff
@@ -211,6 +215,38 @@ func getResourcenameWithTimeHash(prefix string, cluster v1alpha1.Cluster) string
 	return fmt.Sprintf("%s%s%s", prefix, upperClusterID, upperTimeHash)
 }
 
+// imageIDs returns our Container Linux AMIs for each active AWS region. Note
+// that AMIs should always be for HVM virtualisation, not PV. Current Release is
+// CoreOS Container Linux stable 2023.5.0. AMI IDs are copied from the following
+// resource.
+//
+//     https://stable.release.core-os.net/amd64-usr/2023.5.0/coreos_production_ami_hvm.txt.
+//
+func imageIDs() map[string]string {
```

Comment:
That thing is also nicer now. There is also not need anymore to update any useless tests. We do not need to test configuration in such a way.